crates/rolldown/tests/esbuild/importstar/import_default_namespace_combo_issue446/artifacts.snap

```diff
@@ -38,6 +38,30 @@ source: crates/rolldown_testing/src/integration_test.rs
 ```text
 [UNRESOLVED_IMPORT] Warning: "external" is imported by "external-ns.js", but could not be resolved – treating it as an external dependency.
 
+```
+## IMPORT_IS_UNDEFINED
+
+```text
+[IMPORT_IS_UNDEFINED] Warning: Import `def` will always be undefined because there is no matching export in 'internal.js'
```

Comment:
Currently our diagnostic is sub-optimal compared to `esbuild` since we don't store the `span` for each `props` so we could not highlight the target prop accurately, for most scenarios member expr prop will not duplicate.